### ISSUE TITLE
Review content ep. 7

### ISSUE THREAD

**Issue by torwhite** [carpentries-incubator/Reproducible-Publications-with-RStudio#191]
Make sure it makes sense to work through (someone other than Torin)

**Comment by apandas** [comment]
Revisions & Suggested revisions:

Section "Basic Anatomy of the Code Chunk":
I changed "Ctrl + Alt + I (OS X: Cmd + Option + I)" to "Ctrl + Alt + i (OS X: Cmd + Option + i)"

Section "Add a Code Chunk":
I updated instructions. The image <https://ucsbcarpentry.github.io/Reproducible-Publications-with-RStudio/fig/07-heartrate-code.PNG> did not match the instructions. Image shows code chunk in untitled.rmd but instructions had said the code chunk is in DataPaper-ReproducibilityWorkshop.rmd.

I did not commit a change for this. I think we should remove tip that is below the image <https://ucsbcarpentry.github.io/Reproducible-Publications-with-RStudio/fig/07-heartrate-code.PNG>. It repeats information from the beginning of the section. Keep the screenshot bc it is helpful.

Section "Code Chunk Naming and Options"
I tried to fix the rendering for challenge 7.3. wasn't sucessful
